commands.go

```diff
@@ -328,18 +328,6 @@ func initCommands(
 		// Plumbing
 		//-----------------------------------------------------------
 
-		"0.12upgrade": func() (cli.Command, error) {
-			return &command.ZeroTwelveUpgradeCommand{
-				Meta: meta,
-			}, nil
-		},
-
-		"0.13upgrade": func() (cli.Command, error) {
```

Comment:
There's another mention of these in the `HiddenCommands` map later in this file that we could remove.

Reply:
Thank you!